vue/configurator/src/components/Configurator.vue

```diff
@@ -321,4 +409,9 @@ export default class Configurator extends Vue {
   word-wrap: break-word;
   overflow: auto;
 }
+label {
```

Comment:
minor comment: if anyone were to add a `label` element to the configurator, it would be styled using the rules below.